app/Services/PredictionGetter.php

```diff
@@ -0,0 +1,88 @@
+<?php  
+namespace App\Services;
+
+use App\Models\ExchangeRate;
+
+class PredictionGetter {
+
+	public function get() {
```

Comment:
`get` method have to receive $days parameter
```suggestion
	public function get(int $days) {
```